aws/container-linux/kubernetes/network.tf

```diff
@@ -52,6 +52,60 @@ resource "aws_subnet" "public" {
 resource "aws_route_table_association" "public" {
   count = "${length(data.aws_availability_zones.all.names)}"
```

Comment:
define `${length(data.aws_availability_zones.all.names)}` as local

Reply:
This is repeated a few times and I'm looking to minimize the diff here. Similar to another comment I'd like to keep private networking and general refactoring/modifications to typhoon separate.